cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1894,6 +1894,10 @@ if (roofit_multiprocess)
       if(NOT ZeroMQ_FOUND)
         message(STATUS "ZeroMQ not found. Switching on builtin_zeromq option")
         set(builtin_zeromq ON CACHE BOOL "Enabled because ZeroMQ not found (${builtin_zeromq_description})" FORCE)
+        # If the ZeroMQ system version is too old, we can't use the system C++
+        # headers either (note that find_package(ZeroMQ) not only checks if the
+        # library exists, but also if it's a recent version with zmq_ppoll).
+        set(builtin_cppzmq ON CACHE BOOL "Enabled because ZeroMQ not found (${builtin_cppzmq_description})" FORCE)
```

Comment:
It's not the same line, the previous line is for `builtin_zeromq` and this one is for `builtin_cppzmq`.

Yes, actually it would have also been possible to check the `cppzmq` version number, as `zmq.hpp` has some preprocessor macros to indicate the version. But I think it's not worth to do this check, because usually on the system the `cppzmq` bindings come synchronized with `zeromq` and for this we already check the version in `find_package`.

That's why I thought it's simpler to just take the result of `find_package(ZeroMQ)` and enable/disable both `builtin_zeromq` and `builtin_cppzmq` simultaneously according to that.